backendjs/server/api/controllers/group/controller.js

```diff
@@ -109,5 +150,25 @@ export class Controller {
       });
     }
   }
+  async join(req, res) {
+    const user = req.user.sub;
+    const { id } = req.params;
+    try {
+      let participation = await GroupParticipation.create({
```

Comment:
would we want a `GroupParticipation.findOrCreate` and return `400 Bad Request` if already in the group?